### PR TITLE
Added Marketplace extension page.

### PR DESCRIPTION
**Pull request by pccornel**
Added a README-style page for this extension.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Infrastructure (change which improves the lifecycle management (CI/CD, build, package, deploy, lint, etc) of the application, but does not change functionality.)
- [ ] Technical debt (change which improves the maintainability of the codebase, but does not change functionality)
- [ ] Testing (change which modifies or adds test coverage, but does not change functionality)
- [x] Documentation (change which modifies or adds documentation, but does not change functionality)

## Description

Added a README-style page for both the extension's home in the Visual Studio Code Marketplace as well as after users load the extension into VS Code.

## Motivation and Context

Helps customers use this extension without necessarily going to the full documentation elsewhere online.

## Related Issue(s)

None.

## Testing

Viewed the HTML version of the Markdown on GitHub, and tested all of the links.

## Screenshots (if appropriate)

None.

## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] A short description of the change has been added to the **CHANGELOG**

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: resources/marketplace/extension-page.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+## <a id="additional-setup-steps"></a>Additional Setup Steps
+
+After you install the AWS Toolkit for Visual Studio Code, you must complete these additional steps to access most of its features:
+
+1. [Create an AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) and an [access key](https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/).
+1. Connect the Toolkit to AWS by using [AWS credentials](#managing-aws-resources).
+
+To use this Toolkit to develop [serverless applications with AWS](https://aws.amazon.com/serverless/), you must also do the following on the local machine where the Toolkit is installed:
+
+1. Install the [AWS Command Line Interface (AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
```
Comment (awschristou) [inline resources/marketplace/extension-page.md]: Missing close parenthesis: ` (AWS CLI`

File: resources/marketplace/extension-page.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+The Toolkit produces the application code and files, adds them to the location you specified, and opens the program file in the editor.
+
+For more information, see [Creating a Serverless Application](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/create-sam.html) in the _AWS Toolkit for Visual Studio Code User Guide_.
+
+[Return to Top](#top)
+
+#### <a id="run-debug-serverless-application"></a>Run or Debug a Serverless Application
+
+In the **AWS: Explorer** Side Bar, after you choose the **AWS** icon in the Activity Bar, CodeLenses display within open serverless application code files above functions that use AWS Lambda function handler syntax. (A _handler_ is a function that Lambda calls to start execution of a Lambda function.) These CodeLenses enable you to run or debug the corresponding serverless application locally. CodeLens actions here include:
```
Comment (awschristou) [inline resources/marketplace/extension-page.md]: `In the **AWS: Explorer** Side Bar, after you choose the **AWS** icon in the Activity Bar` 
->
'Clicking on the **AWS** icon in the Activity Bar activates the Toolkit. Once the toolkit is activated, CodeLenses display...`

File: resources/marketplace/extension-page.md (hunk 1 above)
Comment (pccornel) [reply resources/marketplace/extension-page.md]: Add missing parenthesis.

File: resources/marketplace/extension-page.md (hunk 2 above)
Comment (pccornel) [reply resources/marketplace/extension-page.md]: Removed "In the AWS: Explorer Side Bar..."

**Pull request merged**